pages.de/common/bird.md

```diff
@@ -0,0 +1,13 @@
+# bird
+
+> BIRD Internet Routing Daemon.
+> Routing-Daemon mit Unterstützung für BGP, OSPF, Babel und weitere.
+> Weitere Informationen: <https://bird.network.cz/>.
+
+- Starte Bird mit einer bestimmten Konfigurationsdatei:
+
+`bird -c {{path/to/bird.conf}}`
+
+- Starte Bird als spezifischer Benutzer und Gruppe:
+
+`bird -u {{username}} -g {{group}}`
```

Comment:
```suggestion
`bird -u {{benutzername}} -g {{group}}`
```

Reply:
Should the arguments all be lowercase or, depending on the context, uppercase? For example, I would capitalize "username" and "group".

Reply:
Not sure tbh for German pafe, will ping @pixelcmtd for further remarks

Reply:
AFAIK, we never capitalize any placeholder, except of course for concrete examples that might have to be capitalized etc